pandas_transform/notebooks/dataframe_example.ipynb

```diff
@@ -0,0 +1,2363 @@
+       "61      39.51         1         NaN           NaN          1.8          4.7  \n",
+       "325     36.48         1         NaN           NaN          4.0          3.6  \n",
+       "337     22.32         1         NaN           NaN          3.9          8.4  \n",
+       "439     35.51         1         NaN           NaN          3.5          7.7  \n",
+       "557     23.84         1         NaN           NaN          1.3          6.8  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "adult_positive = covid_testing.loc[(covid_testing.loc[:,\"result\"] == \"positive\") & (covid_testing.loc[:,\"age\"] > 18), :]\n",
+    "adult_positive.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e95f024-cf64-4b61-99db-301a1999a176",
+   "metadata": {},
+   "source": [
+    "When combining conditions, make use parentheses around each condition, and especially when combining more than two conditions.\n",
+    "\n",
+    "The condition:\n",
+    "`((positive test) & (older than 10)) | (younger than 5)`\n",
+    "is true for all results where a patient was older than 10 and tested positive, as well as all children under five, whether or not they tested positive. By moving the parentheses we can change the condition to test for patients older than 10 or younger than 5, but only return those with positive test results.\n",
+    "\n",
+    "When working with more complicated conditions it is extremely helpful to define your condition by giving it a name, and then refer to that name, rather than the chain of conditions, inside the `.loc` method. By carefully naming the compound condition, you can also make your code more human-readable. Take a moment to see what this code below is doing:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "844b1f34-8f8d-4cf0-87bc-b00a70bbea95",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "mrn                    5007240\n",
+       "first_name               arryk\n",
+       "last_name              mormont\n",
+       "gender                    male\n",
+       "pan_day                     11\n",
+       "test_id                  covid\n",
+       "clinic_name       clinical lab\n",
+       "result                positive\n",
+       "demo_group          misc adult\n",
+       "age                       21.0\n",
+       "drive_thru_ind               1\n",
+       "ct_value                 39.51\n",
+       "orderset                     1\n",
+       "payor_group                NaN\n",
+       "patient_class              NaN\n",
+       "col_rec_tat                1.8\n",
+       "rec_ver_tat                4.7\n",
+       "Name: 61, dtype: object"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "positive_tests.loc[61,:]"
```

Comment:
I'm not sure what purpose this code cell serves... was this supposed to be removed maybe?